workspaces/spectacle/src/index.ts

```diff
@@ -265,6 +270,9 @@ export async function makeSpectacle(opticContext: IOpticContext) {
           notificationsUrl: '',
         };
       },
+      invalidateCaches: async (parent: any, args: any, context: any) => {
+        await reload(context.spectacleContext().opticContext);
+      },
     },
```

Comment:
This isn't getting called anywhere and leaks implementation details to its consumers. I'd propose we don't expose it  unless we find a really good reason for it.